### ISSUE TITLE
Discrepancy in Reproduced Results

### ISSUE THREAD

**Issue by Xiang-Think** [DylanOrange/geal#10]
The reproduced results show that the IoU is higher, while AUC/SIM/MAE are all lower than those in the paper. 

<img width="1122" height="252" alt="Image" src="https://github.com/user-attachments/assets/118230d9-a62b-4cb1-9338-fc3b035425c0" />
<img width="1122" height="252" alt="Image" src="https://github.com/user-attachments/assets/16a0aade-c21f-4a91-bf02-e7b866570074" />

Additionally, there are two significant inconsistencies in the open-source code: first, the 2D branch lacks the Dice loss function; second, from the unexpected keys, it can be seen that during the training of the 3D branch, fuse_level should be false.

<img width="487" height="101" alt="Image" src="https://github.com/user-attachments/assets/816d3154-9754-46cb-b159-3f1fdc724dba" />
<img width="1425" height="26" alt="Image" src="https://github.com/user-attachments/assets/43cbfc2a-aae5-4493-bba2-f55256a11216" />
